packages/eslint-plugin/src/rules/strict-boolean-expressions.ts

```diff
@@ -715,11 +715,16 @@ export default util.createRule<Options, MessageId>({
             context.report({
               node,
               messageId: 'conditionErrorNullableObject',
-              fix: util.getWrappingFixer({
-                sourceCode,
-                node,
-                wrap: code => `${code} != null`,
-              }),
+              suggest: [
+                {
+                  messageId: 'conditionErrorNullableObject',
```

Comment:
[Bug] Ah, suggestion messages should generally to be different from the parent report. You'll want to add a new `messageId` for this case.

Filed https://github.com/eslint-community/eslint-plugin-eslint-plugin/issues/368 to help automate enforcing this.